purchase_stock_sync_schedule_date/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2024 Akretion (https://www.akretion.com).
+# @author Mathieu Delva <mathieu.delva@akretion.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+
+{
+    "name": "Purchase Stock Sync Schedule Date",
+    "version": "14.0.1.1.0",
```

Comment:
First published version
```suggestion
    "version": "14.0.1.0.0",
```